CHANGELOG.md

```diff
@@ -36,9 +35,11 @@ and changes prior to that are (unfortunately) done retrospectively. Critical ite
  - Fix for empty docker namespaces in private repositories
  - Fix Docker environment parsing
  - Revert lolcow easter egg
-
-
-
+ - Fix "Duplicate bootstrap definition key" triggered by comments and blank spaces
+ - Fix for docker permission error when downloading multiple layers
+ - Fix parsing of docker tags and version
+ - Add "$@" to an empty runscript when building from docker
```

Comment:
this is added to any CMD/ENTRYPOINT found when building from Docker, to ensure that any user arguments are passed through (this is default with Docker). If it's an empty runscript, we still default to shell.